core/Tracker/TrackerCodeGenerator.php

```diff
@@ -172,13 +172,23 @@ private function getJavascriptTagOptions($idSite, $mergeSubdomains, $mergeAliasU
         }
         // We need to parse_url to isolate hosts
         $websiteHosts = array();
+        $firstHost = null;
         foreach ($websiteUrls as $site_url) {
             $referrerParsed = parse_url($site_url);
-            $websiteHosts[] = $referrerParsed['host'];
+
+            if (!isset($firstHost)) {
+                $firstHost = $referrerParsed['host'];
+            }
+
+            $url = $referrerParsed['host'];
+            if (!empty($referrerParsed['path'])) {
+                $url .= $referrerParsed['path'];
```

Comment:
If a website URL includes a path we do now append it. Before we only passed domains to `tracker.setDomains()`. The name of the tracker method `setDomains` is not really ideal since one can also append paths to a domain but we thought it's better to leave the name like this for a while